clients/src/main/java/org/apache/kafka/common/internals/Plugin.java

```diff
@@ -60,6 +99,13 @@ private static <T> Map<String, String> tags(String key, T instance) {
         return tags;
     }
 
+    /**
+     * Wrap a list of instances into Plugins.
+     * @param instances the instances to wrap
```

Comment:
ditto I think we should also mention that the instance's class name will be used as the value for the `<code>class</code>` tag.

Reply:
This is already mentioned in the class javadoc